content/blog/my-first-day-at-skyworkz.md

```diff
@@ -0,0 +1,52 @@
+---
+id: blog
+title: My first day at Skyworkz
+description: Stefan reflects on his first day at Skyworkz; visiting the AWS Summit in Amsterdam and looking ahead to Blipz Conference.
+author: Stefan Leenen
+tags:
+  - AWS Summit 2023 Amsterdam
```

Comment:
Given history I'm not expecting us to write another blog about this particular AWS Summit. That would make this particular tag not too useful. More generic tagging (e.g. `Blipz, AWS`) would be my preference.